### PR TITLE
Encode all dashboards

### PR DESCRIPTION
**Pull request by mtojek**
Changes:
* encode all dashboards
* remove build logic
* refresh go.mod
* validate package content

### CODE REVIEW

File: magefile.go [hunk 1]
```diff
@@ -8,42 +8,197 @@ package main
...
 )
 
-func Build() error {
-	err := os.RemoveAll(publicDir)
+type fieldEntry struct {
+	name  string
+	aType string
+}
+
+func Validate() error {
```
Comment (ruflin) [inline magefile.go:31]: Why not use the dry-run in the registry for this?

Comment (mtojek) [reply magefile.go:31]: Does `dry-run` verify required fields?

Comment (ruflin) [reply magefile.go:31]: I would need to check. But if not, we should add the check?

Comment (mtojek) [reply magefile.go:31]: I think we reuse the `dry-run` in all projects. It will become single source of knowledge/validation.

Comment (ruflin) [reply magefile.go:31]: ++

Comment (mtojek) [reply magefile.go:31]: https://github.com/elastic/package-registry/pull/507

**Comment by mtojek** [comment]
I think I will close this one and prepare a new PR with recent changes. This way it might be easier than rebase.

**Pull request closed without merging**